.travis.yml

```diff
@@ -2,37 +2,39 @@ language: cpp
 sudo: required
 dist: trusty
 
-compiler:
-  - gcc
+compiler:  # A trick to construct a build matrix
+  - printf "%s" "3.6" >> ~/PYTHON_VERSION ; gcc
```

Comment:
Is `printf "%s" "3.6" >> ~/PYTHON_VERSION` needed yet?

Reply:
@henry0312 Actually not, this was needed when I constructed 3d-matrix [os, compiler (python version in fact), task]. Now it's easier just to add `PYTHON_VERSION` to the each `env` line (e.g. TASK=gpu METHOD=source PYTHON_VERSION=3.6).

Reply:
@henry0312 I updated build matrix of Travis for better understanding :)

Reply:
looks great!